src/amo/components/AddonBadges/index.js

```diff
@@ -39,7 +39,7 @@ export class AddonBadgesBase extends React.Component<InternalProps> {
     return (
       <div className="AddonBadges">
         {addon.is_recommended && clientApp !== CLIENT_APP_ANDROID ? (
```

Comment:
this made me think - _are_ we going to have promoted add-ons for non-desktop Firefox?  (Probably not a discussion for this pr)

Reply:
I agree both that it's something we should ask, and not related to this PR. I'll add a comment to the PRD doc. Thanks for raising this.